restli-common/src/main/pegasus/com/linkedin/restli/common/ExtensionSchemaAnnotation.pdsc

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "record",
+  "name": "ExtensionSchemaAnnotation",
```

Comment:
I'm thinking we should move ResoureKey annotation definition also here: https://jarvis.corp.linkedin.com/codesearch/result/?path=resource-identity%2Fresource-identity-urn%2Fsrc%2Fmain%2Fpegasus%2Fcom%2Flinkedin%2Fresourceidentity%2Furn&reponame=multiproducts%2Fresource-identity&name=ResourceKey.pdl

As this annotation itself doesn't provide all details needed for the injection. It would be good to keep them both in one place. What do you think?
We can do it in a follow-up PR.

Reply:
Yes, I thought about moving ResourceKey into Pegasus. I just felt it is out of the scope of this PR. I will do it in a follow-up PR.